org.eclipse.bluesky/plugin.xml

```diff
@@ -73,7 +73,15 @@
             path="language-configurations/json/language-configuration.json">
       </languageConfiguration>
    </extension>
-      
+
+   <extension
+         point="org.eclipse.ui.genericeditor.reconcilers">
+      <reconciler
+            class="org.eclipse.bluesky.colors.ColorSymbolReconciler"
+            contentType="org.eclipse.bluesky.json">
```

Comment:
I think it should be `org.eclipse.bluesky.css`

Reply:
ColorSymbolReconciler is used for CSS and JSON both. JSON Language Server supports too color symbol with JSON Schema which defines field with "color" format.

Reply:
Ach yes, I didn't notice second ColorSymbolReconciler for css.